Osiris/Hacks/SkinChanger.cpp

```diff
@@ -85,8 +99,8 @@ static void initializeKits() noexcept
     kitsWeapons.reserve(itemSchema->alternateIcons.numElements);
 
     for (const auto& node : itemSchema->alternateIcons) {
-        const auto encoded = node.key;
-        kitsWeapons.emplace_back(int((encoded & 0xFFFF) >> 2), WeaponId(encoded >> 16), node.value.simpleName.buffer.memory); // https://github.com/perilouswithadollarsign/cstrike15_src/blob/f82112a2388b841d72cb62ca48ab1846dfcc11c8/game/shared/econ/econ_item_schema.cpp#L325-L329
+        if (const auto encoded = node.key; (encoded & 3) == 0)
```

Comment:
love to see you figured out the right way to get it in place of all the other people still using Beaker's method